### PR TITLE
removing redundant getting started docs

### LINKED ISSUE #38: Remove redundant docs
I have about 1-2 days more worth of checks, but after that we can remove redundant docs from oras-py here as they are represented under https://github.com/oras-project/oras-www. I will likely tweak the python page there retrieved from here) so it has links to the more detailed developer docs/notes here.

### PR DESCRIPTION
**Pull request by vsoch**
The main getting started docs are provided (as a single markdown file) here, and that s deployed to oras-www. The remaining 
content here is about contributing / development, and I am also adding links between them so it is easy to find the right one

 - This will close #38 

Signed-off-by: vsoch <vsoch@users.noreply.github.com>

### CODE REVIEW

File: docs/1_python.md
Comment (FeynmanZhou) [inline docs/1_python.md:7]: Do you think it will be better to combine [Oras Python](https://oras-project.github.io/oras-py/) to the [official website](https://oras.land/)? People might be easier to find ORAS-python from a unified documentation portal.

Comment (FeynmanZhou) [reply docs/1_python.md:7]: Just noticed you are working on migrating the docs from oras-py to the ORAS website. Let me know if you need assistance.

File: docs/getting_started/index.rst
Comment (FeynmanZhou) [inline docs/getting_started/index.rst:11]: developer user guide => developer guide?

**Review by FeynmanZhou — COMMENTED** [review]
Added small tips. The overall looks good!

File: docs/1_python.md
Comment (vsoch) [reply docs/1_python.md:7]: That's the entire point of this PR - the file here 1_python.md is grabbed and deployed to the official site! It should be done after these two are merged.

Comment (vsoch) [reply docs/1_python.md:7]: This kills two birds with one stone because the corresponding docs are alongisde the respective code (e.g., as I'm updating the library here I can update them) but then nightly they are grabbed and deployed to oras-www. Here is the page here deployed there: https://oras.land/client_libraries/1_python/ and the PR here is removing the redundant docs from here.

File: docs/getting_started/index.rst
Comment (vsoch) [reply docs/getting_started/index.rst:11]: I meant to say "developer user" - a developer user is not a user of the client, but a developer developing with it. I guess maybe it sounds weird, but in my mind:

- developer: sort of referring to a main maintainer (will likely respond to issues / fix bugs)
- developer user: using _someone else's_ SDK, etc, (will likely open issues / report bugs)
- user: typically someone that is just, for example, running the client (but not developing anything)

**Comment by vsoch** [comment]
Thank you @FeynmanZhou ! Let me know if you have other questions or suggestions for changes.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**
